<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmTofuStats</class>
 <widget class="QDialog" name="FmTofuStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tofu statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Statistics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableStats">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>List of tofu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="tofu::Memo" name="memoTofuList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLabel" name="lbLegend">
       <property name="text">
        <string>* &lt;b&gt;Custom-drawn&lt;/b&gt;: control/format, vector emoji. &lt;b&gt;Good&lt;/b&gt;: have glyph. &lt;b&gt;Tofu&lt;/b&gt;: no glyph.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tofu::Memo</class>
   <extends>QPlainTextEdit</extends>
   <header>FmTofuStats.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
